app/helpers/concerns/foreman_salt/layout_helper_extensions.rb

```diff
@@ -1,20 +0,0 @@
-module ForemanSalt
-  module LayoutHelperExtensions
-    extend ActiveSupport::Concern
-
-    included do
-      alias_method_chain :authorized_associations_permission_name, :salt
```

Comment:
This method was added during 1.10-develop and then replaced by the AssociationAuthorizer, and I'm really not sure how to extend that cleanly.  *sigh*